src/mediaserver.py

```diff
@@ -68,6 +68,11 @@ def chunked(size, source):
     # http headers can only be encoded using latin1
     filename_latin1 = filename.encode("latin1", errors="replace").decode("latin1")
     response.headers["Content-Disposition"] = f'inline; filename="{filename_latin1}"'
+
+    client_cache_duration = int(app.config["MEDIA_CLIENT_CACHE_DURATION"])
+    if client_cache_duration > 0 and not app.config("DEBUG"):
```

Comment:
Use `is_dev_mode()` - that's probably also why the tests are failing

Reply:
It does exist: https://github.com/OpenSlides/openslides-media-service/blob/main/src/config_handling.py#L43

I'm not sure when `app.debug` is set and have doubts that it does work correctly with our dev variable `OPENSLIDES_DEVELOPMENT`. Please add a test for this if you want to keep it this way.

Reply:
I missed the import. Hope it works now.